test/integration-tests/Cargo.toml

```
@@ -31,12 +31,17 @@ key-utils = { path = "../../utils/key-utils" }
 mining_device = { path = "../../roles/test-utils/mining-device" }
 mining_device_sv1 = { path = "../../roles/test-utils/mining-device-sv1" }
 mining_proxy_sv2 = { path = "../../roles/mining-proxy" }
-network_helpers_sv2 = { path = "../../roles/roles-utils/network-helpers", features =["with_buffer_pool"] }
+network_helpers_sv2 = { path = "../../roles/roles-utils/network-helpers", features = ["with_buffer_pool"] }
 pool_sv2 = { path = "../../roles/pool" }
 roles_logic_sv2 = { path = "../../protocols/v2/roles-logic-sv2" }
 stratum-common = { path = "../../common" }
 config-helpers = { path = "../../roles/roles-utils/config-helpers" }
 translator_sv2 = { path = "../../roles/translator" }
+sv1_api = { path = "../../protocols/v1", optional = true }
 
 [lib]
 path = "lib/mod.rs"
+
+[features]
+default = []
```

```suggestion
default = ["sv1"]
```

having to add `--features sv1` to `cargo test` feels a bit counter-intuitive... and forgetting to do it basically disables the tests that use Sv1 Sniffer

---

as discussed in a call with @jbesraa , we still only have one single test where we assert any kind of Sv1 behavior

and including the `sv1` feature flag as default would bloat the ITF build process for every other test

so for now, it's ok to leave this as not being a default feature